workflows/qc/make-qc-reference.wdl

```diff
@@ -25,12 +25,62 @@
 
 version 1.0
 
-import "../../tools/fastq_screen.wdl"
+import "../../tools/util.wdl"
+import "../../tools/kraken.wdl"
 
 workflow make_qc_reference {
-    call fastq_screen.build_db as fastq_screen_build_db
+    input {
+        Array[String] kraken_libraries = [
+            "archaea",
+            "bacteria",
+            "plasmid",
+            "viral",
+            "human",
+            "fungi",
+            "protozoa",
+            "UniVec_Core"
+        ]
+        Array[String] fasta_urls = []
+        Array[File] fastas = []
+        Int? max_retries
+    }
+
+    parameter_meta {
+        kraken_libraries: "List of kraken libraries to download. Valid options: `archaea`, `bacteria`, `plasmid`, `viral`, `human`, `fungi`, `plant`, `protozoa`, `nt`, `UniVec`, `UniVec_Core`"
```

Comment:
Should we just link to the manual? https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown#custom-databases

Reply:
I think the manual could be linked the in the header. But I don't think it belongs in the parameter_meta

Reply:
Should this also use the `choices` format?